config/v1/types_infrastructure.go

```diff
@@ -614,6 +625,15 @@ type OpenStackPlatformStatus struct {
 	// datacenter DNS, a DNS service is hosted as a static pod to serve those hostnames
 	// to the nodes in the cluster.
 	NodeDNSIP string `json:"nodeDNSIP,omitempty"`
+
+	// loadBalancerType defines if a internal or external load balancer
+	// is to be used for the openshift api and ingress endpoints.
+	// When set to Internal the static pods in charge of API and Ingress traffic load-balancing
+	// defined in machine config operator will be deployed.
+	// When set to External these static pods will not be deployed.
+	// When not set, the default value is left to the cluster and currently defaults
+	// to Internal, so the static pods will be deployed.
```

Comment:
We have a standard wording for this context, it's something like
```suggestion
	// When omitted, this means no opinion and the platform is left to choose a reasonable default.
	// The current default value is Internal.
```